keystone_api/apps/allocations/tasks.py

```diff
@@ -0,0 +1,184 @@
+"""Schedule tasks executed in parallel by Celery."""
+
+from datetime import date
+import logging
+import re
+from shlex import split
+from subprocess import PIPE, Popen
+from typing import Collection, List
+
+from celery import shared_task
+from django.db.models import Sum, ObjectDoesNotExist
+
+from apps.allocations.models import Allocation, Cluster
+from apps.users.models import ResearchGroup
+
+log = logging.getLogger(__name__)
+
+
+@shared_task()
+def update_limits() -> None:
+    """Adjust per Slurm account usage limits on all enabled clusters"""
+    log.info(f"{date.today()} - BEGIN update_limits")
```

Comment:
The log utility adds the timestamp automatically. Including the date is unnecessary. Also, make the message more descriptive. Don't assume the person going through the logs knows al the function names and what those functions do.